ldapadmin/src/main/webapp/console/app/components/user/user.html

```diff
@@ -67,8 +142,9 @@ <h4 translate>group.system</h4>
 
           <h4 translate>group.app</h4>
           <div class="form-group">
-            <select class="form-control" multiple chosen ng-model="user.user.groups" 
-              ng-options="group for group in user.groups" data-placeholder="Select groups">
+            <select class="form-control" multiple chosen ng-options="group.cn for (key, group) in user.groups"
+            ng-model="user.toto" data-placeholder="Select groups">
```

Comment:
Unexpected change. Compare with [?w=0](https://github.com/tonio/georchestra/pull/1/files?w=0) to see diff without whitespace.